src/services/conversationsService.js

```diff
@@ -25,6 +25,20 @@ const fetchConversations = async function(options) {
 	return axios.get(generateOcsUrl('apps/spreed/api/v4/room'), options)
 }
 
+/**
+ * fetch future events for a given conversation within the next 31 days.
+ *
+ * @param {string} location room's absolute url
+ * @param {object} options options
+ */
+const getUpcomingEvents = async (location, options = {}) => {
+	return axios.get(generateOcsUrl('/apps/dav/api/v1/events/upcoming'), Object.assign(options, {
+		params: {
+			location,
+		},
+	}))
+}
```

Comment:
What `options` can be present here?

Reply:
For now, none just kept there for future expansion 😇 ---> I will remove it